kotest-assertions/src/commonMain/kotlin/io/kotest/properties/Gen.kt

```diff
@@ -28,63 +28,67 @@ import io.kotest.properties.shrinking.Shrinker
  * The [Int] generator example should return a random int
  * from across the entire integer range.
  */
+@Deprecated("Deprecated and will be removed in 5.0. Migrate to the new property test classes in 4.0")
```

Comment:
I would create a PR to remove it with "[5.0 only]" and keep it there. We'll forget about this

Reply:
I think the text will be changed before we go live anyway, as I don't want to keep this around 5.0. Maybe remove in 4.2.